crates/holochain/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ## Unreleased
 
 - Avoid issuing duplicate warrants for invalid actions during app validation by checking for existing warrants before creating new ones. #5352
+- Fix: Make sure peers that were marked unresponsive are not marked responsive again until their agent info expiration timestamp or a new agent info is received.
```

Comment:
Really minor thing, please add new entries to the top. The file is chronological so more recent changes should stack on top.

Reply:
Ah always assumed the opposite. Fixed: https://github.com/holochain/holochain/pull/5423/commits/cdfe24212a5ca82c20c24fb00e23c41e8ba91f0b